.github/workflows/ci.yml

```diff
@@ -10,11 +10,34 @@ jobs:
     runs-on: ${{ matrix.platform }}
     steps:
     - uses: actions/checkout@v1
+    - name: Cache Memgraph Docker image
+      id: cache-memgraph-docker
+      uses: actions/cache@v1
+      with:
+        path: ~/memgraph
+        key: cache-memgraph-v1.0.0-docker-image
```

Comment:
Why is `v1.0.0` used here, but `v1.1.0` is used below?

Reply:
Replaced both with v1.1.0